### ISSUE TITLE
Some 2D items float above the ground

### ISSUE THREAD

**Issue by Zkyo42** [Draylar/better-dropped-items#8]
I've noticed a minor issue some dropped items will hover slightly above the ground. I haven't tested everything, but they all seem to be items that have a 2d sprite, but can be placed in the world as a block. Saplings, seeds, carrots, potatoes, vines, flowers, double flowers, doors, signs, grass, rails, glass panes, iron bars, ladders, and probably several more items hover.

![2020-10-17_19 09 23](https://user-images.githubusercontent.com/69995225/96355276-c8d94180-10ad-11eb-915f-855d26232e2b.png)

**Comment by skylorbeck** [comment]
The cause of this is all of these items are actually an AliasedBlockItem

**Comment by skylorbeck** [comment]
Solved
![image](https://user-images.githubusercontent.com/42103122/98430211-e112ff80-2060-11eb-82c2-45b13a3f06dd.png)

```// offset rendered stacks vertically
        // 0.01 barely brings flat items flush to the ground
        matrixStack.translate(0, 0, 0.01f);
        if(itemEntity.getStack().getItem() instanceof AliasedBlockItem) {

        } else if(itemEntity.getStack().getItem() instanceof BlockItem) {
            // make blocks flush with the ground
            matrixStack.translate(0, 0, -0.12f);
        }
```

**Comment by Draylar** [comment]
Fixed in 1.2.3+ thanks to Skylortrexler.
